src/DiagnosticMessages.ts

```diff
@@ -605,6 +605,16 @@ export let DiagnosticMessages = {
         message: `Missing exception expression after 'throw' keyword`,
         code: 1118,
         severity: DiagnosticSeverity.Error
+    }),
+    callableHasUnknownReturnType: (returnType: string) => ({
```

Comment:
I'm not sure these new diagnostics need to be added - they're a lot like other ones ("invalidFunctionReturnType" and "expectedValidTypeToFollowAsKeyword")

Reply:
If possible yes reuse existing errors if they are fitting